search/releases/7.1/index.adoc

```diff
@@ -58,3 +68,61 @@ List<Book> hits = searchSession.search( Book.class )
     .filter( f.match().field( "authors.firstName" ).matching( "arthur" ) )
 ).fetchHits( 20 );
 ----
+
+By its nature a `knn` predicate will always try to find nearest vectors,
+even if the found vectors are quite far away from each other, i.e. are not that similar.
+This may lead to getting irrelevant results returned by the query.
+
+To address this, a `knn` predicate allows link:{hsearch-doc-url-prefix}#search-dsl-predicate-knn-with-similarity[configuring the minimum required similarity]:
+
+[source, Java, indent=0, subs="+attributes"]
+----
+float[] coverImageEmbeddingsVector = /*...*/
+
+List<Book> hits = searchSession.search( Book.class )
+    .where( f ->
+        // Create a knn predicate as usual:
+        f.knn( 5 ).field( "coverImageEmbeddings" ).matching( coverImageEmbeddingsVector )
+        // Specify the required minimum similarity value, to filter out irrelevant results:
+        .requiredMinimumSimilarity( 5 ) )
+    .fetchHits( 20 );
+----
```

Comment:
I wouldn't mention this in the "What's new" section on the website, as this looks like a relatively advanced feature to me? I'd only mention it in the blog post.
YMMV.

Reply:
That makes sense 😃 I'll remove this one, thanks!